.cicd/create-docker-from-binary.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# buildkite-agent artifact download someversion.deb .
+
+buildkite-agent artifact download '*.deb' --step ':ubuntu: Ubuntu 18.04 - Package Builder' .
+echo ":done: download successfull"
+
+#PREFIX='base-ubuntu-18.04'
+SANITIZED_BRANCH=$(echo "$BUILDKITE_BRANCH" | sed 's.^/..' | sed 's/[:/]/_/g')
+SANITIZED_TAG=$(echo "$BUILDKITE_TAG" | sed 's.^/..' | tr '/' '_')
+echo "$SANITIZED_BRANCH"
+echo "$SANITIZED_TAG"
+
+# do docker build
+echo ":docker::build: Building image..."
+
+IMAGE_ECR="$EOSIO_REGISTRY:$SANITIZED_BRANCH"
+DOCKERHUB_EOS_REGISTRY="docker.io/eosio/eos"
+IMAGE_DOCKER="$DOCKERHUB_EOS_REGISTRY:$SANITIZED_BRANCH"
+
+echo ":docker: Building Image...."
+docker build -t $IMAGE_ECR -t $IMAGE_DOCKER -f ./docker/dockerfile .
+echo "Build Done successfully!!"
+
+#do docker push to ECR
+echo "Pushing Image to ECR..."
+docker push $IMAGE_ECR
+echo ":done: Done"
+
+# do docker push Dockerhub
+echo "Pushing Image DockerHub EOSIO/EOS ..."
+docker push $IMAGE_DOCKER
```

Comment:
In addition to tagging and pushing using the branch, we should also be tagging and pushing using the commit and tag (if present), as described in [AUTO-313](https://blockone.atlassian.net/browse/AUTO-313). The commit will be the primary way internal systems pull these images by (to guarantee you are getting the specific version of software under test), while the tag will be the primary way external consumers pull their images (to get an image containing a specific release).

Reply:
I implemented the tag using the Branch, commit and tag (if exists) to following the recommendation and looking in the examples scripts

Reply:
Unfortunately, since `BUILDKITE_COMMIT` cannot be known in advance, the schema being used for tagged images...
```
$EOSIO_REGISTRY:$SANITIZED_BRANCH-$BUILDKITE_COMMIT-$SANITIZED_TAG
```
...is also not predictable. We want people (and systems) to be able to pull down a container with the tagged build in it without having to know what the commit hash corresponding to that tag is. Because of this, all builds should push with commit...
```
$EOSIO_REGISTRY:$BUILDKITE_COMMIT
```
...branch...
```
$EOSIO_REGISTRY:$SANITIZED_BRANCH
```
...and tagged builds should be pushing a third time with the `git` tag, if this is different than the `git` branch:
```
$EOSIO_REGISTRY:$SANITIZED_TAG
```
This way, you can pull the container corresponding to any `git` reference you want, without having to know all three references. The examples provided in the ticket work this way. You can copy from them, or we can hop on video and work more closely on it.

Reply:
Done, I combined the recommendations made by Scott